fmriprep/interfaces/maths.py

```diff
@@ -0,0 +1,44 @@
+import os
+from nipype.interfaces.base import SimpleInterface, TraitedSpec, traits, File
+from nipype.utils.filemanip import fname_presuffix
+
+
+class ThresholdInputSpec(TraitedSpec):
+    in_file = File(exists=True, mandatory=True, desc="Input imaging file")
+    out_file = File(desc="Output file name")
+    threshold = traits.Float(0., usedefault=True, desc="Values under this become 0")
+
+
+class ThresholdOutputSpec(TraitedSpec):
+    out_file = File(desc="Output file name")
+
+
+class Threshold(SimpleInterface):
```

Comment:
I think if we are having a `Threshold` interface, it needs to handle upper/lower bounds. Alternatively, this could be named `LowerThreshold` instead to avoid any confusion.